src/sql/azdata.d.ts

```diff
@@ -4645,15 +4645,10 @@ declare module 'azdata' {
 		}
 
 		export interface INotebookMetadata {
-			kernelspec: IKernelInfo;
+			kernelspec?: IKernelSpec;
 			language_info?: ILanguageInfo;
 			tags?: string[];
-		}
-
-		export interface IKernelInfo {
-			name: string;
-			language?: string;
-			display_name?: string;
+			azdata_notebook_guid?: string;
```

Comment:
This wasn't really something we wanted to expose to users - we should see if there's a way not to have this in the public api